modules/op-release-notes-1-16.adoc

```diff
@@ -643,3 +643,15 @@ With this update, {pipelines-title} General Availability (GA) 1.16.2 is availabl
 === Fixed issues
 
 * Before this update, in {pipelines-shortname} 1.16, you could not cancel a pipeline run by patching the `PipelineRun` object and setting the `spec.status` parameter to `Cancelled` if the first task run in the pipeline run was completed. Instead, an error message was logged: `PipelineRun was cancelled but had errors trying to cancel the TaskRuns and/or Runs`. With this update, the pipeline run is cancelled successfully.
+
+[id="release-notes-1-16-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.16.3
+
+With this update, {pipelines-title} General Availability (GA) 1.16.3 is available on {OCP} 4.15 and later versions.
+
+[id="fixed-issues-1-16-3_{context}"]
+=== Fixed issues
+
+* Before this update, in some cases the {tekton-chains} controller repeatedly crashed, making the {tekton-chains} component unusable. With this update, the controller no longer crashes.
+
+* Before this update, if you defined a matrix task that included both regular parameters and `matrix` parameters, the `tekton-pipelines-controller` component crashed and logged a segmentation fault message. If the `Task` custom resource (CR) was not removed, the component continued to crash and did not run any pipelines. With this update, the controller no longer crashes in such cases.
```

Comment:
It is not clear to me how these two sentences are related. If this release note is correct, feel free to ignore my ignorant concerns.